dist/images/ovnkube.sh

```diff
@@ -352,54 +348,53 @@ process_healthy () {
 # checks for the health of the process either using `ps` or `ovs-appctl`
 # $1 is the name of the process
 # $2 is the process pid
-check_health () {
+check_health() {
   ctl_file=""
   case ${1} in
-    "ovnkube"|"ovnkube-master")
-    ;;
-    "ovnnb_db"|"ovnsb_db")
+  "ovnkube" | "ovnkube-master") ;;
```

Comment:
This change seems weird to me. It makes more sense to me to always have the `;;` on its own line

Reply:
ok moved to its own line.